src/App.jsx

```diff
@@ -14,33 +16,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
+const REVERSE_SORT = 'reverse';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [sortBy, setSortBy] = useState('');
+  const [reversed, setReversed] = useState('');
+  const [lightlyClass, setLightlyClass] = useState('');
+  const [lightlyClassReversed, setlightlyClassReversed] = useState(false);
+  let [...visibleGoods] = goodsFromServer;
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  visibleGoods = visibleGoods.sort((good1, good2) => {
```

Comment:
Instead of this variable you can create a function that takes goods, sortBy and reversed states and based on them will return a new sorted array